IaC/AKS/cluster/charts/fluentbit/templates/config.yaml

```diff
@@ -57,23 +57,30 @@ data:
         K8S-Logging.Exclude Off
         Annotations         Off
 
+    [FILTER]
+        Name    modify
+        Match   kube.*
+        Copy    kubernetes_labels_app_kubernetes_io_instance   k_app
+        Copy    kubernetes_labels_app                          k_app
+        Copy    kubernetes_container_name                      k_container
```

Comment:
I think this is what you were referring to in the Teams chat, but wanted to clarify.  Only one of kubernetes_labels_app_kubernetes_io_instance or kubernetes_labels_app will exist depending on how things are running? Mainly checking to make sure the k_app field doesn't get overwritten in any scenario.

Reply:
yes - I updated the chat with more details